frontends/concrete-python/examples/sha1/sha1.md

```diff
@@ -0,0 +1,307 @@
+# SHA1 Computation with Modules
```

Comment:
SHA1 computation with modules

Reply:
Sure? All our "main titles" in https://docs.zama.ai/concrete/tutorials/see-all-tutorials are with Capitalization on All Laters.

Reply:
In general, we follow the [Google Developer Documentation Style Guide](https://developers.google.com/style/capitalization, so we only use sentence-case capitalization in tech docs. (The title-case capitalization is more more marketing blogs, PR, etc.)

Reply:
so do you want me to change the other tutorials?

Reply:
Sure if you can!
But otherwise no worries, I can do it later.

Reply:
ok done